lib/thrift/transport/ssl.ex

```diff
@@ -0,0 +1,46 @@
+defmodule Thrift.Transport.SSL do
+  @moduledoc """
+  SSL configuration helpers.
+
+  Clients and servers support SSL with the `:ssl_opts` start_link option. There are additional options from `:ssl` but
+  otherwise configuration is the same:
```

Comment:
Perhaps we should add a link in here to the [Erlang ssl module documentation](http://erlang.org/doc/man/ssl.html) for quick access when people are reading these docs.

Reply:
Good point, I'll add a link, sorry I merged this before your review Michael.